### PR TITLE
[HB-5106] Rename Package & Contents to Chartboost Mediation

### PR DESCRIPTION
**Pull request by SCastanedaMunoz**
# Description

- Renaming com.chartboost.helium to com.chartboost.mediation
- Renaming existing references to Helium locations to Chartboost Mediation
- Renaming Documetation from Helium to Chartboost Mediation
- Updating Dependencies from Helium to Chartboost Mediation

### CODE REVIEW

**Review by pleasesavemycat — APPROVED** [review]
lgtm

**Review by kushG — APPROVED** [review]

File: com.chartboost.mediation.demo/Assets/Plugins/Android/mainTemplate.gradle [hunk 1]
```diff
@@ -0,0 +1,137 @@
+// Android Resolver Repos Start
+([rootProject] + (rootProject.subprojects as List)).each { project ->
+    project.repositories {
+        def unityProjectPath = $/file:///**DIR_UNITYPROJECT**/$.replace("\\", "/")
+        maven {
+            url "https://maven.google.com"
+        }
+        maven {
+            url "https://cboost.jfrog.io/artifactory/helium" // Assets/Samples/Chartboost Mediation/4.0.0/Chartboost Mediation/Editor/ChartboostMediationDependencies.xml:7
```
Comment (cb-jpadilla) [inline com.chartboost.mediation.demo/Assets/Plugins/Android/mainTemplate.gradle:9]: Small fyi: this repo will change too. But, it should be ok for now until the new one is created.

File: com.chartboost.mediation.demo/Assets/Plugins/Android/mainTemplate.gradle [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+      exclude ('/lib/x86_64/*' + '*')
+  }
+}
+// Android Resolver Exclusions End
+android {
+    compileSdkVersion **APIVERSION**
+    buildToolsVersion '**BUILDTOOLS**'
+
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
```
Comment (cb-jpadilla) [inline com.chartboost.mediation.demo/Assets/Plugins/Android/mainTemplate.gradle:111]: 🤔 .... this is still needed on Unity, isn't?? I wonder when they will start using Java 11.

File: com.chartboost.mediation.demo/Assets/Samples/Chartboost Mediation/4.0.0/Chartboost Monetization/Editor/Optional-ChartboostDependencies.xml [hunk 3]
```diff
@@ -5,7 +5,7 @@
         <androidPackage spec="com.chartboost:helium-adapter-chartboost:4.9.1.1+@aar" />
 
         <!-- Helium Chartboost SDK -->
```
Comment (cb-jpadilla) [inline com.chartboost.mediation.demo/Assets/Samples/Chartboost Mediation/4.0.0/Chartboost Monetization/Editor/Optional-ChartboostDependencies.xml:7]:
```suggestion
        <!-- Chartboost Monetization SDK -->
```
We may need to also rename the rest, don't we? 🤔

File: com.chartboost.mediation/Editor/HeliumIntegrationChecker.cs [hunk 4]
```diff
@@ -187,27 +187,28 @@ public static void CheckUnityAdsIntegration()
...
         /// <summary>
-        /// Used to update all existing adapters by Devs choice. This will utilize current's Helium Package version to override all adapters with such version.
+        /// Used to update all existing adapters by Devs choice. This will utilize current's Chartboost Mediation Package version to override all adapters with such version.
         /// </summary>
-        [MenuItem("Helium/Integration/Force Reimport Adapters", false, 2)]
+        [MenuItem("Chartboost Mediation/Integration/Force Reimport Adapters", false, 2)]
         public static void ForceReimportExistingAdapters()
         {
-            var confirmUpdate = EditorUtility.DisplayDialog(HeliumWindowTitle,
+            var confirmUpdate = EditorUtility.DisplayDialog(WindowTitle,
```
Comment (cb-jpadilla) [inline com.chartboost.mediation/Editor/HeliumIntegrationChecker.cs:211]: Can this name collide with any files from 3rd-parties?? Would `ChartboostMediationWindowTitle` help??

**Review by cb-jpadilla — COMMENTED** [review]
Mostly looks good. Only ask: Can we remove the .DS_Store file and can we add it to the .gitignore?

**Pull request merged**